github/files/drupal7/scripts/devops/reinstall/pre_settings.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Adding database variable to settings.php
   sudo: yes
-  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"127.0.0.1\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
+  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"{{ mysql_host }}\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
```

Comment:
Can we provide a more recent workaround that is created previously?
Like we have pre_settings array of data that should be inserted to the settings.php file https://github.com/propeoplemd/cibox/blob/master/github/files/drupal7/scripts/devops/reinstall/vars/environments/global_env.yml#L4 before Db initialized.
and real life example 
![image](https://cloud.githubusercontent.com/assets/563412/11121452/9b9d5a5c-895e-11e5-9913-06d0fafb2d0b.png)
/cc @Sanchiz @danylevskyi